libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/ElementTheme.kt

```diff
@@ -46,20 +51,26 @@ object ElementTheme {
 
 /* Global variables (application level) */
 val LocalColors = staticCompositionLocalOf { elementColorsLight() }
+val LocalCompoundColors = staticCompositionLocalOf { compoundColorsLight }
```

Comment:
It is more ergonomic if we surface this API as an extension over `MaterialTheme` and hide the composition locals (i.e. make them internal/private implementation details).

Access to these colors could then be made with code such as: `MaterialTheme.compoundColors`

draft of impl:
```
val MaterialTheme.compoundColors
    @Composable
    get() = LocalCompoundColors.current
```